priv/repo/seeds.exs

```diff
@@ -18,7 +18,23 @@ import Openmaize.DB
 company1 = Repo.insert! %Company{
   name: "Hársnyrtifélagið Gæðahár",
   phone: "+354 823 9745",
-  email: "harsnyrtifelagid@gmail.com"
+  email: "harsnyrtifelagid@gmail.com",
+  description: "Hársnyrtifélagið is one of Reykjavíks hidden gems when it comes to hair.",
+  website_url: "https://news.ycombinator.com/",
+  facebook: "harsnyrtifelagid",
+  twitter: "@harsnyrtifelagid",
+  location_name: "Kramhúsið",
+  location_street: "Skólavörðustígur 19",
+  location_city: "Reykjavík",
+  location_zip: "101",
+  location_country: "IS",
+  opening_hours: %{
+    "monday" => ["9-17"],
```

Comment:
Hehe yeah so this is absolutely not how we want the format to be. My plan for this PR was to not cover the validation of the `opening_hours` field, but obviously this PR should do that. I'll add it tomorrow. 

The question that remains is still how we want the format to be, and the main question there is probably whether we want to support multiple opening ranges per day, like: `{ "monday": [{9, 12}, {13, 17}] }`. What do you think?